.github/workflows/release-embedding-sdk.yml

```diff
@@ -37,8 +37,138 @@ jobs:
           triggered by: @${{ github.event.sender.login }}
           EOF
 
-  determine-version:
-    name: Determine SDK version
```

Comment:
renamed to `determine-sdk-version` to match the other job which doesn't have `name:`, but have descriptive names.